<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Криптография</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnHiSqr">
      <property name="text">
       <string>Тестирование скремблера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnXor">
      <property name="text">
       <string>XOR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHill">
      <property name="text">
       <string>Шифр Хилла. Демонстрация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFeistel">
      <property name="text">
       <string>Сеть Фейстеля</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDes">
      <property name="text">
       <string>DES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGenPrime">
      <property name="text">
       <string>Генерация простых чисел заданной длины</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrimitiveRoots">
      <property name="text">
       <string>Поиск первых 100 первообразных корней числа</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDiffieHellman">
      <property name="text">
       <string>Демонстрация протокола Диффи-Хеллмана</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRSA">
      <property name="text">
       <string>RSA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHash">
      <property name="text">
       <string>ЭЦП</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
